server/README.md

```diff
@@ -31,7 +35,7 @@ The INS server provides a backend RESTful service for INS-node and front-end web
     #### Example
   * INS-node with device id 1000 triggers device position computation after sending several RSSI readings.
 
-    `http://localhost:9080/resolve_pos/1000`
+    `http://localhost:9080/resolve_pos/5239`
```

Comment:
What was wrong with `id 1000`? :stuck_out_tongue_closed_eyes:

Reply:
same as before. Just keeping the commands easy for copy and paste during debugging.

Reply:
If you must keep it as `5239` then please update the documentation two lines above!